lib/Config.js

```diff
@@ -155,41 +172,51 @@ class Config {
                     'bad config: utapi workers must be a positive integer');
                 this.utapi.workers = config.utapi.workers;
             }
-            if (config.utapi.redis) {
-                if (config.utapi.redis.sentinels) {
-                    this.utapi.redis = { sentinels: [], name: null };
+            // `localCache` is not part of the utapi configuration property.
+            assert(config.localCache, 'missing required property of utapi ' +
```

Comment:
Since we are expecting localCache from S3' config and not in the Utapi section
`config.localcache: missing required property of utapi: localCache` .
The comment above should be updated as `Utapi uses the same localCache config defined for S3 to avoid config duplication`